src/utils/colors.ts

```diff
@@ -0,0 +1,6 @@
+export const Colorwhite = '#ffffff';
```

Comment:
This way to use and declare the colors is not correct. 

it should be something like this: 
```
export const colors = {
   white: '#ffffff',
   black: '#000000',
   gray: .....
}

```
and used like this: 

```
import colors from '../utils/colors'

.....
background-color: colors.white

```